styles/style.css

```diff
@@ -66,12 +66,31 @@ body {
 .jumbotron {
 	border-radius: 0;
 }
+container {
+	display: flex;
+	width: 100%;
+	height: auto;
+	flex-direction: row;
+	
```

Comment:
this class is missing a `.` at the beginning - but aside from this, be careful: `.container` is already a Bootstrap class so overwriting it can cause unexpected behaviour.